docs/user/managing-devices.md

```diff
@@ -485,6 +495,60 @@ LABEL appType="compose"
 
 ```
 
+### Embedding Applications
```

Comment:
Would thia be more suitable as sub-section of “Image Building”?

Reply:
will try to cover both